modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -122,7 +131,6 @@ class Numeric extends Component {
 
 	_internalValueFn() {
 		const {value} = this;
-
```

Comment:
@alinedoleron Not sure we have a rule for that, but I prefer a line between variable declarations and the return statement.